Annotato/Annotato/Views/Document/Edit/DocumentView.swift

```diff
@@ -70,6 +70,17 @@ class DocumentView: UIView {
                 self?.replaceSelectionBox(newSelectionBoxFrame: newSelectionBoxFrame)
             }
         }).store(in: &cancellables)
+
+        viewModel.$connectivityChanged.sink(receiveValue: { [weak self] _ in
+            // This means that the document view model has changed the annotations in
+            // the document, so we re-render again
+            guard let self = self else {
+                return
+            }
+            self.removeAllAnnotationsFromView()
+            self.annotationViews = []
+            self.initializeInitialAnnotationViews()
+        }).store(in: &cancellables)
```

Comment:
Do we really need all of these? I thought we are just going to call `viewWillAppear` in `DocumentEditViewController` when the user disconnects?

Reply:
Ok that works too
Then it will automatically reload all the annotations again right?

Reply:
Yes

Reply:
Ok got it, I changed